### PR TITLE
d8 gent-base integration of infobox

### PR DESCRIPTION
**Pull request by hstandaert**
## Description

Display infobox on: 
- Events
- News
- Infopoages

## How Has This Been Tested?

Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: templates/custom/vesta/vesta--teaser-with-link.html.twig [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * @see template_preprocess_vesta()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<article{{ attributes.addClass('teaser-content') }}>
+  <div class="content__bottom">
+    <h4 id="{{ entity.id }}" class="teaser-title-link">
+      <a href="{{ url }}" class="standalone-link">{{ entity.name }}</a>
```
Comment (delrueba) [inline templates/custom/vesta/vesta--teaser-with-link.html.twig]: No links in titles.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: gent_base.theme [hunk 2]
```diff
@@ -402,11 +402,12 @@ function gent_base_preprocess_field__field_contacts(&$variables) {
   if (count($contacts) > 2) {
     foreach ($contacts as $key => $contact) {
       $variables["items"][$key]["content"]["#view_mode"] = 'teaser_tiny';
-      $variables['contacts'][] = $variables["items"][$key];
     }
   }
   else {
-    $variables['contacts'] = $variables["items"];
+    foreach ($contacts as $key => $contact) {
+      $variables["items"][$key]['attributes']->addClass('teaser', 'teaser-contact');
```
Comment (MPParsley) [inline gent_base.theme]:
```suggestion
      $variables['items'][$key]['attributes']->addClass('teaser', 'teaser-contact');
```

File: gent_base.theme [hunk 3]
```diff
@@ -419,6 +420,23 @@ function gent_base_preprocess_field__field_responsible_person(&$variables) {
...
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_field__field_contact_categories(&$variables) {
+  $contacts = $variables['element']['#object']->field_contacts->count();
+  $contact_categories = $variables['items'];
+  if ($contacts <= 2) {
+    foreach ($contact_categories as $key => $category) {
+      $variables["items"][$key]['attributes']->addClass('teaser', 'teaser-contact');
```
Comment (MPParsley) [inline gent_base.theme]:
```suggestion
      $variables['items'][$key]['attributes']->addClass('teaser', 'teaser-contact');
```

File: gent_base.theme [hunk 4]
```diff
@@ -419,6 +420,23 @@ function gent_base_preprocess_field__field_responsible_person(&$variables) {
...
+    }
+  }
+
+  foreach ($variables['items'] as $key => $item) {
+    $variables["items"][$key]["content"]["#options"]['attributes']['class'][] = 'standalone-link';
```
Comment (MPParsley) [inline gent_base.theme]:
```suggestion
    $variables['items'][$key]['content']['#options']['attributes']['class'][] = 'standalone-link';
```

**Review by MPParsley — APPROVED** [review]

File: templates/custom/vesta/vesta--teaser-with-link.html.twig (hunk 1 above)
Comment (meirege) [reply templates/custom/vesta/vesta--teaser-with-link.html.twig]: 👍

**Review by delrueba — APPROVED** [review]

**Pull request merged**